ui/src/userInput/showInputBox.ts

```diff
@@ -40,9 +44,14 @@ export async function showInputBox(context: IInternalActionContext, options: Inp
                     inputBox.enabled = true;
                     inputBox.busy = false;
                 }),
-                inputBox.onDidTriggerButton(_btn => {
-                    // Only back button is supported for now
-                    reject(new GoBackError());
+                inputBox.onDidTriggerButton(async btn => {
+                    if (btn === QuickInputButtons.Back) {
+                        reject(new GoBackError());
+                    } else if (btn === AzExtQuickInputButtons.LearnMore) {
+                        await openUrl(nonNullProp(options, 'learnMoreLink'));
+                        context.telemetry.properties.learnMoreStep = context.telemetry.properties.lastStep;
+                        context.telemetry.properties.clickedLearnMore = 'true';
```

Comment:
We don't need `clickedLearnMore` because we know it was clicked if `learnMoreStep` is defined